lib/ruby-debug-ide/attach/util.rb

```diff
@@ -1,5 +1,16 @@
 require 'ruby-debug-ide/attach/lldb'
 require 'ruby-debug-ide/attach/gdb'
+require 'socket'
+
+def get_child_pids(pid, pids = Array.new)
+  pids << pid
+  pipe = IO.popen("pgrep -P #{pid}")
```

Comment:
for MacOS > 10.8 works fine. For previous versions it is necessary to install `protocols`
https://unix.stackexchange.com/questions/225/pgrep-and-pkill-alternatives-on-mac-os-x